### ISSUE TITLE
android minSDK level

### ISSUE THREAD

**Issue by hansimmmi** [ionspin/kotlin-multiplatform-libsodium#31]
Hi! @ionspin 
Thank you always for your prompt support. Just a question regarding android minimum SDK level. 
The library requires minSDK level of 24, is there a specific reason for the minSDK level? Could it be lower to 21 instead?
That would make this library be more widely accepted.

**Comment by ionspin** [comment]
Hi @hansimmmi,

There was a technical reason why I went with 24 specifically, but at the moment I can't remember why. 

The other reason is that even level 24 is now 6 years old (, and has probably stopped receiving any kind of security updates. My general plan was to bump up the minSDK level as time passes, but I haven't had a system to decide when to bump the version so I haven't done it yet.

To sum things up, I won't be bringing the minSDK level down, sorry.
